sdk/monitor/Azure.Monitor.Query/tests/MetricsQueryClientSamples.cs

```diff
@@ -307,5 +307,47 @@ public void CreateClientsWithOptions()
                 logsQueryClientOptions);
             #endregion
         }
+
+        [Test]
+        public async Task QueryResourcesMetricsWithOptionsStartTimeEndTime()
```

Comment:
Are there existing samples using the TimeRange property that we should update or delete?

Reply:
No we never had it in the read me